src/main/java/io/appium/java_client/gecko/GeckoDriver.java

```diff
@@ -84,20 +133,24 @@ public GeckoDriver(HttpClient.Factory httpClientFactory, Capabilities capabiliti
      * ClientConfig clientConfig = ClientConfig.defaultConfig()
      *     .baseUri(URI.create("WebDriver URL"))
      *     .readTimeout(Duration.ofMinutes(5));
+     * AppiumClientConfig appiumClientConfig = AppiumClientConfig.defaultConfig()
+     *     .setHttpClientConfig(clientConfig).directConnect(true);
      * GeckoOptions options = new GeckoOptions();
-     * GeckoDriver driver = new GeckoDriver(clientConfig, options);
+     * GeckoDriver driver = new GeckoDriver(options, appiumClientConfig);
      *
      * </pre>
      *
-     * @param clientConfig take a look at {@link ClientConfig}
      * @param capabilities take a look at {@link Capabilities}
+     * @param appiumClientConfig take a look at {@link AppiumClientConfig}
      *
      */
-    public GeckoDriver(ClientConfig clientConfig, Capabilities capabilities) {
```

Comment:
Please do not change the original sequence of arguments, because this would be a breaking change then

Reply:
Also AppiumClientConfig must be inherited from ClientConfig in order to make this contructor backward compatible

Reply:
Yes, the clientConfig version has not been released yet, so I modified it. But this will stay later.